app/src/main/java/com/github/fribourgsdp/radio/SetUtility.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.fribourgsdp.radio
+
+object SetUtility {
+    /**
+     * Add an item to a set, after removing it if it was present
+     */
+    fun <E> addToSet(set : MutableSet<E>, item : E){
```

Comment:
Isn't there a function or class that does the same thing in the Kotlin documentation?

Reply:
Unfortunately not, we have to remove and then add, and as we do this frequently, it was to avoid code duplication that I created this static class/object.